### DISCUSSION TITLE
Tab Islands like in Opera

### DISCUSSION THREAD

**Discussion by drugpup** [Floorp-Projects/Floorp#803] [Feature Request]
I'm asking if it would be possible to integrated Tab islands similar to the opera browser. (Link to the Opera blog-post "https://blogs.opera.com/news/2023/06/opera-tab-islands/". It would Help with organization, productivity and would provide simple ease of use and feature expansion.

**Comment by mahcin** [comment]
Only if feature could be switched on/off by user and not permanent as because of tab islands I moved out of Opera (about 3000pages at the session I have). Different people, different needs

**Reply by drugpup** [reply]
Ofcourse, I never intended to suggest to force the feature onto everyone.

**Comment by Git-North** [comment]
[Tab Islands](https://www.opera.com/tr/features/tab-islands#:~:text=Tab%20Islands%20is%20such%20a,are%20naturally%20grouped%20by%20context.) is a feature I fell in love with immediately. And something I would love to see in Floorp

**Comment by AxeldeWater** [comment]
This feature is the top requested feature requested by Firefox's users from Mozilla. It has been for about 2 years, but Mozilla doesn't seem to make steps towards fulfilling this request. It would be nice if Floorp somehow managed to add this feature.

**Reply by AxeldeWater** [reply]
https://connect.mozilla.org/t5/ideas/native-tab-grouping-more-customizable-tab-bar/idi-p/303

**Comment by CutterKnife** [comment]
Tab Groups like Chrome had been considered, but were abandoned due to technical issues.
There was a move to replace Workspaces v1 with Tab Stacking like Vivaldi, but due to multiple issues, it was replaced with Workspaces v2.

They know that there are so many requests.

**Reply by AxeldeWater** [reply]
Would it be because the whole backend of the tab system should be redesigned for support of grouped tabs?

**Reply by CutterKnife** [reply]
Perhaps, yes. I think they said there was a problem with the tab management API.
